src/h5web/guards.ts

```diff
@@ -145,11 +141,9 @@ export function assertGroup(
   }
 }
 
-export function assertReachable(
-  link: HDF5Link
-): asserts link is HDF5HardLink | HDF5RootLink {
-  if (!isReachable(link)) {
-    throw new Error('Expected link to be reachable');
+export function assertHardLink(link: HDF5Link): asserts link is HDF5HardLink {
+  if (!isHardLink(link)) {
+    throw new Error('Expected link to be hard link');
```

Comment:
Or simply _Expected a hard link_ ?

Reply:
This function will be gone in my next PR 😉